deb/brave-keyring/DEBIAN/postinst

```diff
@@ -2,3 +2,11 @@
 
 # Delete (expired) key added by old install instructions
 apt-key --keyring /etc/apt/trusted.gpg del D8BAD4DE7EE17AF52A834B2D0BB75829C2D4E821
+
+# Match if the new keyring in /usr/share/keyrings is in use.
+new_keyring_pattern='^[[:space:]]*deb[^#]*signed-by=/usr/share/keyrings/brave-browser-release.asc[^#]*brave-browser-apt-release.s3.brave.com'
+
+# Delete the old keyring file if the new one is already in use.
+if grep -q "${new_keyring_pattern:?}" /etc/apt/sources.list /etc/apt/sources.list.d/*; then
```

Comment:
Is there a better way to do this?

Reply:
I'm not aware of a better way to do it.

Reply:
For simplicity, I would suggest only looking at `/etc/apt/sources.list.d/brave-browser*` since that's what our instructions use.

Users not following our instructions get to cleanup this old keyring on their own :)